backends-common/rabbitmq/src/main/java/org/apache/james/backend/rabbitmq/RabbitMQConnectionFactory.java

```diff
@@ -58,8 +58,8 @@ Connection create() {
     }
 
     Mono<Connection> connectionMono() {
+        Duration forever = Duration.ofMillis(Long.MAX_VALUE);
```

Comment:
it should be extracted into a constant, and somewhere else

Reply:
For what? Is forever ever going to change?
Are we going to use this FOREVER for any other purpose? Are we going to reference it every time we call retryBackoff? And what if we fail? Is it a problem?
In my opinion, defining this constant would require a time module or a reactor module somewhere and we would then include it everywhere. And then, we would have to know that we should not use the reactor code directly but use this thin wrapper we have. It looks like something I don't want to do for now for a single constant that will likely never change.

Reply:
I just meant, extract it into a private constant in that class to save a `Duration.ofMillis(Long.MAX_VALUE)` call each time calling connectionMono()

Reply:
Let's keep it simple for now, I'm pretty sure it will never be a performance problem.